contributors/guide/github-workflow.md

```diff
@@ -220,7 +220,7 @@ fork.
 
 Very small PRs are easy to review.  Very large PRs are very difficult to review.
 At the assigned reviewer's discretion, a PR may be switched to use
```

Comment:
Please remove everything in this paragraph other than `Very small PRs are easy to review.  Very large PRs are very difficult to review.`. We don't use reviewable at all anymore.

Reply:
this is done.thanks for the review